src/analyses/ai_storage.h

```diff
@@ -168,12 +168,12 @@ class location_sensitive_storaget : public trace_map_storaget
   }
 
   cstate_ptrt abstract_state_before(
-    locationt t,
+    locationt l,
```

Comment:
Please squash with the commit that introduces this.

Reply:
I tried; it was a mess.  I will try again when doing a final rebase and squash of revision comments.